packages/office-ui-fabric-react/src/components/ComboBox/examples/ComboBox.Basic.Example.tsx

```diff
@@ -213,17 +229,17 @@ export class ComboBoxBasicExample extends React.Component<any, any> {
 
   @autobind
   private _onChanged(option: IComboBoxOption, index: number, value: string) {
-    if (option !== undefined) {
+    if (option !== null) {

```

Comment:
This is due to merge complication. I didn't intend to change these. I'll revert these back.